jamaljsr/polar
# Add right-click menus for nodes and channels

### Description

This PR adds right-click context menus for nodes & channels to save a few clicks getting to the actions. This just surfaces the same actions that are available in each node's sidebar Actions tab. You can also right click on the backend connections to change the backend of an LN node.

### Steps to Test

1. Create a network and start it
1. Right-click on the any node
1. Confirm you can see the actions listed
1. Click on one or more of them to confirm the correct action is triggered
1. Open a channel
1. Right-click on the channel
1. Confirm you can see the "Close Channel" action listed

### Screenshots

![image](https://user-images.githubusercontent.com/1356600/75743414-f1f3c000-5cde-11ea-9eb3-99cd19cf99cf.png)

![image](https://user-images.githubusercontent.com/1356600/75743425-fa4bfb00-5cde-11ea-8528-4bcdf8135e29.png)

![image](https://user-images.githubusercontent.com/1356600/75743438-020b9f80-5cdf-11ea-80da-30438e2d28cb.png)

![image](https://user-images.githubusercontent.com/1356600/75743448-089a1700-5cdf-11ea-80a9-664bd6743989.png)
